addons/l10n_in/models/ir_attachment.py

```diff
@@ -0,0 +1,27 @@
+from odoo import models, api
+import json
+
+
+class IrAttachment(models.Model):
+    _inherit = 'ir.attachment'
+
+    @api.model
+    def create_json_attachment(self, name, data, model, record_id, company_id=None):
```

Comment:
```suggestion
    def _l10n_in_generate_json_attachment(self, name, data, model, record_id, company_id=None):
```
Suggest renaming the method to _l10n_in_generate_json_attachment for better clarity, 
as it aligns with Indian localization naming and clearly indicates that the method generates a JSON attachment.